src/pkg/backup/details/details_test.go

```diff
@@ -224,6 +225,63 @@ var pathItemsTable = []struct {
 		expectRepoRefs:     []string{"abcde", "12345"},
 		expectLocationRefs: []string{"locationref", "locationref2"},
 	},
+	{
+		name: "multiple entries with meta file",
+		ents: []DetailsEntry{
+			{
+				RepoRef:     "abcde",
```

Comment:
It might not affect this test, but we should always have a suffix of `.data`, `.meta` or `.dirmeta` for the locationRef for any newer backups.

Reply:
Let's not cross wires, and come back to that in a different PR.

Reply:
What I meant by newer backups is that any backups done with the latest version of the code.

Reply:
Understood.  What I mean is that this test can be brought up to date in a different PR.

Also, now that I'm re-reading your comment, `locationRef` should _not_ contain those values.  It's entirely a folder reference, and contains no file data.  It should be unaware of both data and meta categorization.